media/server/gstplayer/source/tasks/generic/SetPlaybackRate.cpp

```diff
@@ -81,6 +83,15 @@ void SetPlaybackRate::execute() const
         RIALTO_SERVER_LOG_DEBUG("Sent new segment, success = %s", success ? "true" : "false");
         m_gstWrapper->gstSegmentFree(segment);
     }
+#if GST_CHECK_VERSION(1, 18, 0)
```

Comment:
If m_enableInstantRateChangeSeek is enabled but not supported is it worth logging a warning?

Reply:
warning added